src/components/Example/Example.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { ExampleWrapper } from './Example.styles';
+import { File, Heart, ArrowCircleLeft, HeartFull, Share } from '../Icons';
 
 const Example = ({ value }) => {
   return (
     <ExampleWrapper>
       <p>{value}</p>
+      <File />
+      <Heart />
+      <HeartFull />
+      <ArrowCircleLeft />
+      <Share />
     </ExampleWrapper>
   );
 };
```

Comment:
Remove the icons from Example, otherwise we'll have merge conflicts.

Reply:
All good